light_curves/code_src/wise_functions.py

```diff
@@ -225,7 +225,10 @@ def load_lightcurves(locations, radius, bandlist):
         match_df = pixel_tbl.take(pixel_ilocs).to_pandas()
         # attach identifying info like the objectid by joining with locs_df
         match_df["object_ilocs"] = object_ilocs
-        match_df = match_df.set_index("object_ilocs").join(locs_df.reset_index(drop=True))
+        match_df = match_df.set_index("object_ilocs").join(
+            locs_df.reset_index(drop=True),
+            rsuffix="_obj"             # prevents ra/dec collisions
```

Comment:
```suggestion
            rsuffix="_obj"  # prevent name collisions if sample_table has extra columns
```

A collision can only happen if the user added an extra column(s) to `sample_table`, and then it could be with any column we load from WISE and not just ra or dec.